PhpAmqpLib/Helper/DebugHelper.php

```diff
@@ -22,6 +22,9 @@ class DebugHelper
      * @param string $PROTOCOL_CONSTANTS_CLASS
      */
     public function __construct($PROTOCOL_CONSTANTS_CLASS) {
+        if(!defined('STDOUT')) define('STDOUT', fopen('php://stdout', 'w'));
+        if(!defined('STDERR')) define('STDERR', fopen('php://stderr', 'w'));
```

Comment:
Can you please remove `STDERR` as well? It's not used and doesn't make sense here.